duckduckgo/privacy-dashboard
# Malicious website protection - Windows support

Asana: https://app.asana.com/0/72649045549333/1208745889841028/f
**Reviewer:** @shakyShane 

Note: Includes https://github.com/duckduckgo/privacy-dashboard/pull/252

## Description:

Enables Malicious site protection warnings on Windows

## Steps to test this PR:

1. Visit [preview](https://deploy-preview-282--ddgdash.netlify.app/app-debug/html/iframe?state=malware&platforms=windows)
2. Confirm that both phishing and malware states show on Windows
